Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apt-get update \
 		openjdk-11-jre openssh-client perl-modules python3 python3-requests \
 		make patch repo sudo texinfo vim-tiny wget whiptail libelf-dev git-lfs screen \
 		socket corkscrew curl xz-utils tcl libtinfo5 device-tree-compiler python3-pip python3-dev \
+		tmux libncurses5-dev vim \
```

Comment:
I think we can replace libncurses5-dev with just libncurses-dev. From ubuntu:

```
libncurses-dev - developer's libraries for ncurses
libncurses5-dev - transitional package for libncurses-dev
```

libncurses5-dev is just a transitional package, which depends on libncurses-dev.

Can you validate with libncurses-dev instead?

Also, don't worry about doing another push as it will create another CI run for changes not related to lmp directly, so once you're able to test with libncurses-dev and are happy with it, push directly to master.

Reply:
thanks a lot